### PR TITLE
Ref fill user card

### PR DESCRIPTION
**Pull request by marinarodriguez**
Some functions have been changed to arrow functions on `fill-user-card.js`, the others are called before being declared.

Also some issues have been solved:
-Default image is not longer deleted from the previews when the page is reloaded without adding an image in the form section.
-When reloading with some cache information, if the user hasn't filled the name in the form, a default one is applied in the preview.
-When reloading with some cache information, if the user hasn't filled the job in the form, a default one is applied in the preview.

### CODE REVIEW

File: _src/assets/js/fill-user-card.js [hunk 1]
```diff
@@ -54,25 +54,25 @@ function reloadPage() {
...
     linkedinCardEl.href = `https://www.linkedin.com/in/${userData.l}`;
-    linkedinInputEl.value = userData.linkedin
+    linkedinInputEl.value = userData.linkedin;
 
-    githubEl.value = userData.github
+    githubEl.value = userData.github;
     githubLinkEl.href = `https://github.com/${userData.github}`;
     themeUser();
-    emptyImg();
+    //emptyImg();
```
Comment (tanialopezab) [inline _src/assets/js/fill-user-card.js]: Is this comment strictly necessary? @marinarodriguez

**Review by tanialopezab — CHANGES_REQUESTED** [review]
I leave you a comment on filll-user-card.js on line 75 about a commented line. Please have a look

**Comment by marinarodriguez** [comment]
Comment deleted! it wasn't needed indeed.

**Comment by tanialopezab** [comment]
ok all its ok. You can merge

**Review by tanialopezab — APPROVED** [review]

**Pull request merged**